addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -146,8 +147,26 @@ def _compute_plan_id(self):
                 scheduler.plan_id = False
 
     @api.depends('res_model_id', 'res_ids')
-    def _compute_plan_date_deadline(self):
-        self.plan_date_deadline = False
+    def _compute_plan_date(self):
+        self.plan_date = False
+
+    @api.depends('plan_date', 'plan_id')
+    def _compute_plan_summary(self):
+        for scheduler in self:
+            template_ids = scheduler.plan_id.template_ids
+            if not template_ids:
+                scheduler.plan_summary = False
+                continue
+            summaries = []
+            for template in template_ids:
```

Comment:
Why not like the original code ?
```suggestion
    def _compute_plan_summary(self):
        self.plan_summary = False
        for scheduler in self.filtered('template_ids'):
            summaries = []
            for template in template_ids:
```